modules/installation/pages/general-requirements.adoc

```diff
@@ -56,7 +57,7 @@ You do not require a separate code for SLES{nbsp}{sles-version} {sp-version}.
 endif::[]
 
 If not already done, download the {sle} Unified Installer from https://download.suse.com/index.jsp.
```

Comment:
```suggestion
If not already done, download the {sle} Unified Installer from https://download.suse.com.
```
Works fine, and we don't care if the index is jsp, php, htmp, py or whatever else :-D

In the end that's working just as a redirect.